src/plugins/telemetry/server/routes/telemetry_usage_stats.ts

```diff
@@ -31,9 +35,22 @@ export function registerTelemetryUsageStatsRoutes(
     async (context, req, res) => {
       const { unencrypted, refreshCache } = req.body;
 
+      const security = getSecurity();
+      if (security && unencrypted) {
+        // Normally we would use `options: { tags: ['access:decryptedTelemetry'] }` in the route definition to check authorization for an
+        // API action, however, we want to check this conditionally based on the `unencrypted` parameter. In this case we need to use the
+        // security API directly to check privileges for this action. Note that the 'decryptedTelemetry' API privilege string is only
+        // granted to users that have "Global All" or "Global Read" privileges in Kibana.
+        const { checkPrivilegesWithRequest, actions } = security.authz;
+        const privileges = { kibana: actions.api.get('decryptedTelemetry') };
+        const { hasAllRequested } = await checkPrivilegesWithRequest(req).globally(privileges);
```

Comment:
Do we have FTR coverage on this endpoint? If so it may be good to add a test for this behavior.

Reply:
Done in [`b50da6b` (#126238)](https://github.com/elastic/kibana/pull/126238/commits/b50da6bb14b3d5b7b1d6e6a9eee986f28a8b00f8)